tasks/sips/args/sipArgs.ts

```diff
@@ -402,6 +402,50 @@ const sip0071 = async (hre: HardhatRuntimeEnvironment): Promise<ISipArgument> =>
     };
 };
 
+const sipSOV3564 = async (hre: HardhatRuntimeEnvironment): Promise<ISipArgument> => {
+    const {
+        ethers,
+        deployments: { get },
+    } = hre;
+
+    // @todo for the mainnet deployment first run `yarn deploy --tags 'StabilityPool,BorrowerOperations,TroveManager' --network rskSovrynMainnet`
+    const newStabilityPoolImplementation = (await get("StabilityPool_Implementation")).address;
+    const newBorrowerOperationsImplementation = (await get("BorrowerOperations_Implementation")).address;
+    const newTroveManagerImplementation = (await get("TroveManager_Implementation")).address;
+
```

Comment:
pls add validation of current deployments - they should differ from the current logics used by the proxies.